src/NuGet.Core/NuGet.Test.Utility/TestPackages.cs

```diff
@@ -539,70 +543,70 @@ public static Stream GetTestPackageWithContentXmlFile()
             return stream;
         }
 
-        public static FileInfo GetPackageWithSHA512AtRoot(string path, string packageId, string packageVersion)
+        public static async Task<FileInfo> GetPackageWithSHA512AtRoot(string path, string packageId, string packageVersion)
```

Comment:
there is not much point in making tests fully async, it forces all consumers to go async for no strong reason